platform.md

```diff
@@ -304,9 +306,11 @@ Usage:
 | `<gid>`           | `CNB_GROUP_ID`        |                          | Primary GID of the build image `User`
 | `<layers>`        | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
 | `<image>`         |                       |                          | Tag reference to which the app image will be written
+| `<launch-cache>`  | `CNB_LAUNCH_CACHE_DIR`|                          | Path to a cache directory containing launch layers
 | `<log-level>`     | `CNB_LOG_LEVEL`       | `info`                   | Log Level
 | `<previous-image>`| `CNB_PREVIOUS_IMAGE`  | `<image>`                | Image reference to be analyzed (usually the result of the previous build)
 | `<run-image>`     | `CNB_RUN_IMAGE`       | resolved from `<stack>`  | Run image reference
+| `<skip-layers>`   | `CNB_SKIP_LAYERS`     | `false`                  | Do not restore SBOM layer from previous image
```

Comment:
Why do we split this across two phases?
1. Restoring the sbom directory during `analyze`
2. Copying sbom files to layers directories and then deleting sbom directory during `restore`

Wouldn't it be cleaner just to do everything during the restore phase so that the exact steps (restore, copy, delete) can be implementation details rather that specified behavior. Also, `restorer` already receives the `skip-layers` flag, so we would only need to add a single flag `-launch-cache` to the restore phase.

Reply:
We were torn between these two options (see https://github.com/buildpacks/spec/issues/276#issuecomment-992712735). Having restore do everything would require additional logic to provide the `-daemon` flag, mount in the daemon socket, and run as root.